apm-agent-plugins/apm-jms-plugin/src/main/java/co/elastic/apm/agent/jms/JmsMessageConsumerInstrumentation.java

```diff
@@ -237,6 +245,9 @@ public static void afterReceive(@Advice.Origin Class<?> clazz,
                         messageHandlingTransaction.appendToName(" from ");
                         helper.addDestinationDetails(message, destination, destinationName, messageHandlingTransaction);
                         helper.addMessageDetails(message, messageHandlingTransaction);
+                        if (messageAge > 0) {
```

Comment:
same as previous, shouldn't it be `>= 0` ?

Reply:
same :)